setup.py

```diff
@@ -23,8 +23,22 @@
         pinnedVersion = line.split()[0]
         install_requires.append(pinnedVersion)
 
+dependency_links = []
+try:
+    with open("constraints.txt") as constraintsFile:
+        for line in constraintsFile:
+            line = line.strip()
+            if len(line) == 0:
```

Comment:
Don't think it's necessary to check for an empty line. Calling find on an empty string returns -1 so it should be fine with the last check.